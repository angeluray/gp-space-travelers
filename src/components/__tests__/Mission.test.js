import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import MyMission from '../Mission/Mission ';
import store from '../../redux/configureStore';

it('Test Missions renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <MyMission />
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
