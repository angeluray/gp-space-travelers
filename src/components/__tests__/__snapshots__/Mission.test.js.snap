// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Missions renders correctly 1`] = `
<div
  className="container"
>
  <table
    className="container my-5 table table-lg table-striped table-bordered table-hover"
  >
    <thead>
      <tr>
        <th>
          Mission
        </th>
        <th>
          Description
        </th>
        <th>
          Status
        </th>
      </tr>
    </thead>
    <tbody />
  </table>
</div>
`;

exports[`Test RocketContainer renders correctly 1`] = `
<div
  className="container"
>
  <ul />
</div>
`;
